### ISSUE TITLE
windows fig startup & autocomplete doesn't work

### ISSUE THREAD

**Issue by cpreston321** [withfig/fig#2181]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

Windows 11 Pro

### Expected behaviour

Just works on startup like MAC OS

### Actual behaviour

Starts a terminal then sits and waits, I then run `fig launch` it says Fig already running. Then I wait then Fig app shows up. I changed a couple of settings then freezes close out, then I can never open it again until restarting PC. Also Autocomplete doesn't seem to work at all.  

### Steps to reproduce

Just install with specs then seems to reproduce, I tried uninstalling and reinstalling. Nothing seems to fix it.

### Environment

```yaml
fig-details:
  - cli-version: 1.5.0
  - desktop-version: 2.5.3
  - figterm-version: 5.2.0
hardware-info:
  - chip-id: AMD Ryzen 9 7950X 16-Core Processor
  - cores: 16
  - mem: 31892.043 GB
os-info:
  - Windows 10 Pro (or newer) - build 22621
environment:
  - shell: C:\Program Files\Git\usr\bin\bash.exe
  - terminal: <unknown>
  - cwd: C:\Users\chris
  - exe-path: C:\Users\chris\AppData\Local\Fig\latest\fig.exe
  - install-method: unknown
  - env-vars:
    - DISPLAY: needs-to-be-defined
    - FIG_INTEGRATION_VERSION: 8
    - FIG_PID: 740
    - FIG_TERM: 1
    - FIG_TERM_VERSION: 5.2.0
    - FIG_WORKFLOWS_KEYBIND: ^f
    - PATH: C:\Users\chris\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\local\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\chris\bin;C:\Python310\Scripts;C:\Python310;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\nodejs;C:\ProgramData\chocolatey\bin;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\Git\cmd;C:\Users\chris\AppData\Local\pnpm;C:\Users\chris\AppData\Local\Microsoft\WindowsApps;C:\Users\chris\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\chris\AppData\Roaming\npm;C:\Users\chris\AppData\Local\Fig\latest;C:\Program Files\Git\usr\bin\vendor_perl;C:\Program Files\Git\usr\bin\core_perl;C:\Users\chris\.local\bin;C:\Users\chris\.fig\bin
    - SHELL: C:\Program Files\Git\usr\bin\bash.exe
    - TERM: xterm-256color
    - TERM_SESSION_ID: df4ac241-df0f-4a4e-9fe8-9765a7dffaae
```
```
